airbyte-integrations/connectors/source-snapchat-marketing/source_snapchat_marketing/spec.json

```diff
@@ -45,6 +45,30 @@
         "examples": ["2022-01-30"],
         "order": 4,
         "format": "date"
+      },
+      "action_report_time": {
+        "type": "string",
+        "enum": ["conversion", "impression"],
+        "title": "Action Report Time",
+        "description": "Specifies the principle for conversion reporting",
```

Comment:
Please add dots at the end of descriptions.

Reply:
Done